source/docs/software/advanced-gradlerio/profiling-with-visualvm.rst

```diff
@@ -100,7 +100,7 @@ To begin, ensure you are on the :guilabel:`Monitor` tab and click :guilabel:`Hea
    :alt: Location of heap dump button in VisualVM
    :width: 700
 
-This heap dump will be stored on the target system (roboRIO) and must be retrieved using SFTP. See :doc:`this article </docs/software/roborio-info/roborio-ftp>` for information on retrieving the dump from the roboRIO.
+This heap dump will be stored on the target system (SystemCore) and must be retrieved using SFTP. See :doc:`this article </docs/software/roborio-info/roborio-ftp>` for information on retrieving the dump from the SystemCore.
```

Comment:
Assuming these changes are correct, are they in scope of this PR?

Given that these pages are in a directory named after the RIO, perhaps a separate PR should be done to update both content and directory structure?

Reply:
Yes, the ftp document is going to need a bigger update, but that should be done separately. The visualvm article should be fine with these minor changes.